### PR TITLE
Modify Search and Rescue behavior

### PR DESCRIPTION
**Pull request by rodan123**
Turning in a search and rescue item should pass the same variables as a commodity sell to VoiceAttack.
Changed code to send item to VA as _EDDI search and rescue commodity_.
 Standalone script was only using the rare flag in addition to the name, not worth denying the item name to VA for that. If we really need the rare flag, it should be sent as a separate variable.
Modified standalone script, removed _Humanise()_ from reward, it was making the sentence nonsensical at times.

### CODE REVIEW

**Comment by Tkael** [comment]
Why not make both the raw name and the object available? The name could be sent to VA under 'EDDI search and rescue name' while still preserving the information that is made available from the current object.

Please provide an example - how was Humanise() making the reward nonsensical?

**Comment by Tkael** [comment]
Firstly, for backwards compatibility. Changing the variable from event.commodity.name to event.commodity is an easily avoidable script breaking change for everyone using the current script.

Secondly, as a philosophy, we generally try to make as much information available at possible, even if it isn't used by the default script (power users may find uses we didn't anticipate)

Regarding Humanise(), I'd like to keep that in as it generally improves the flow of the dialog, but also create a second pull request with your proposed change to that function.

**Comment by rodan123** [comment]
You make a good point about not breaking scripts.  I have reworked the code to provide an additional variable and leave the commodity object as it is. I named the variable for VA as _EDDI search and rescue commodityname_ , a little wordy, but that way it closely matches the commodity object's _commodity.name_ which it mirrors. Will test and post up. I reverted all the changes to the S&R cottle script, except for the single typo correction, so it won't conflict with any changes you make. Let me know if you want me to do the new pull request for altering the "Humanize" function.

**Comment by Tkael** [comment]
Yes please do. I'd like to give the team a chance to weigh in on the proposed change to Humanise() and a separate PR is a good way to manage that.

File: speechresponder/eddi.json [hunk 1]
```diff
@@ -1275,7 +1275,7 @@
...
...s\", \"black boxes\")}\r\n   |else: \r\n      {OneOf(\"flight record\", \"black box\")}\r\n   }\r\n\r\n|elif event.commodity.name = \"Occupied CryoPod\":\r\n   {OneOf(\"damaged cryo pod\", \"occupied cryo pod\")}{if event.amount > 1:s}\r\n\r\n|elif event.commodity.name = \"Occupied Escape Pod\":\r\n   {OneOf(\"damaged escape pod\", \"occupied escape pod\")}{if event.amount > 1:s}\r\n\r\n|elif event.commodity.name = \"Personal Effects\":\r\n   {OneOf(\"set\", \"collection\")}{if event.amount > 1:s} of\r\n   {OneOf(\"personal effects\", \"personal belongings\")}\r\n\r\n|elif event.commodity.name = \"Salvageable Wreckage\":\r\n   {set wreckage to OneOf(\"wreckage\", \"debris\")}\r\n   {set fragment to OneOf(\"fragment\", \"bit\", \"piece\", \"part\", \"remnant\")}\r\n   {OneOf(\"{fragment}{if event.amount > 1:s} of {wreckage}\", \r\n          \"{wreckage} {fragment}{if event.amount > 1:s}\")}\r\n\r\n|elif event.commodity.name = \"Hostage\":\r\n   {OneOf(\"hostage\")}{if event.amount > 1:s}\r\n\r\n|else:\r\n   {event.commodity.name}{if event.amount > 1:s}\r\n}\r\n\r\n{OneOf(\"recovered\", \"salvaged\")}\r\n{Occasionally(2, \"in exchange\")}\r\nfor {Humanise(event.reward)} credits\r\n",
```
Comment (Tkael) [inline speechresponder/eddi.json]: Please revert this change to prevent a merge conflict with PR #263 (since both would otherwise modify the same file). I've incorporated the correction into that PR.

**Comment by rodan123** [comment]
OK - Think it is good to go, now.

**Comment by Tkael** [comment]
Hmm. It jumped from two files modified to four, and the diff is now including additional items.
Would it be possible to resubmit a cleaned version of this PR with only the changes to Events/SearchAndRescueEvent.cs and a description of the change in EDDI/ChangeLog.md?

**Comment by rodan123** [comment]
Sure no problem. Closing this pull request.
